doc/classes/InputEvent.xml

```diff
@@ -64,27 +65,28 @@
 			<argument index="0" name="action" type="String">
 			</argument>
 			<description>
-				Returns true if the given action is released (i.e. not pressed). Not relevant for the event types MOUSE_MOTION, SCREEN_DRAG and NONE.
+				Returns [code]true[/code] if the given action is released (i.e. not pressed). Not relevant for the event types MOUSE_MOTION, SCREEN_DRAG and NONE.
 			</description>
 		</method>
 		<method name="is_action_type" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
+				Returns [code]true[/code] if current input event has a type, a concret implementation and not only the main [code]InputEvent[/code] class.
 			</description>
 		</method>
 		<method name="is_echo" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
-				Returns true if this input event is an echo event (only for events of type KEY, it will return false for other types).
+				Returns [code]true[/code] if this input event is an echo event (only for events of type KEY, it will return false for other types).
 			</description>
 		</method>
 		<method name="is_pressed" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
-				Returns true if this input event is pressed. Not relevant for the event types MOUSE_MOTION, SCREEN_DRAG and NONE.
+				Returns [code]true[/code] if this input event is pressed. Not relevant for the event types MOUSE_MOTION, SCREEN_DRAG and NONE.
```

Comment:
Add [code] tags to constant names. 

Missing Oxford comma:
types MOUSE_MOTION, SCREEN_DRAG, or NONE.